css/style.css

```diff
@@ -1,4 +1,38 @@
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,700);
+/*
+ * Muli is licensed under SIL OFL Version 1.1
+ * See /assets/logos/font/Muli/OFL.txt file for full information.
+ */
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Bold.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Regular.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Light.woff') format('woff');
+    font-weight: 300;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Italic.woff') format('woff');
```

Comment:
Bold-and-Italic format is missing, mind adding it if you have it?

Reply:
Is that used somewhere in docs?

Reply:
I'm not sure now, but we should really add it, otherwise the first `strong` within an italic text (or the other way around) is going to look very ugly. I don't think it's expensive to add it and pretty standard to ship the 4 main combinations.